System-Automation-Scripts/Automating_Google.py

```diff
@@ -0,0 +1,38 @@
+# In this script we will automate Google using Selenium 
+
+# Imports : 
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+import time
+from time import sleep
+from webdriver_manager.chrome import ChromeDriverManager
+ 
+driver = webdriver.Chrome(ChromeDriverManager().install())
+
+# enter the link of the google
+driver.get("http://www.google.com")
```

Comment:
Webdriver prioritize `https` more than `http`. Please update the following changes in the parse link!